File: problems/src/clojure/src/misc/fizzbuzz.clj
(ns problems.misc.fizzbuzz)

(defn fizzbuzz [n]
  (cond (= 0 (mod n 15)) (prn "fizzbuzz")
        (= 0 (mod n 3)) (prn "fizz")
        (= 0 (mod n 5)) (prn "buzz")))

(map fizzbuzz (range 1 101))

File: problems/src/java/src/udemy/FizzBuzz.java
package udemy;

public class FizzBuzz {

    public static void main(String[] args) {
        FizzBuzz fb = new FizzBuzz();
        fb.fizzbuzz(100);
    }

    public void fizzbuzz(int n) {
        StringBuilder out = new StringBuilder();
        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0 && i % 5 == 0)
                out.append("FizzBuzz");
            else if (i % 3 == 0)
                out.append("Fizz");
            else if (i % 5 == 0)
                out.append("Buzz");
            else
                out.append(i);
            System.out.println(out.toString());
            out.delete(0, out.length());
        }

    }
}
